features/FEATURE_COMMON_PAL/mbed-coap/.settings/language.settings.xml

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Thanks for the clarification. I didn't realize its intended to be present in the development repo.